shipitfile.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (shipit) => {
+  require('shipit-deploy')(shipit);
+
+  shipit.initConfig({
+    default: {
+      workspace: '.',
+      deployTo: process.env.DEPLOY_SERVICE_PATH,
+      // don't send node_modules of packages since they've been bundled,
+      // send root node_modules since we need them to start the app.
+      ignores: ['packages/**/node_modules'],
+      keepReleases: 5,
+      keepWorkspace: true,
+      shallowClone: false,
+      branch: 'HEAD',
+    },
+    production: {
+      servers: {
+        user: process.env.HOST_HK_1_USER,
+        host: process.env.HOST_HK_1_IP,
+      },
+    },
+  });
+
+  shipit.on('published', async () => {
+    shipit.start('server:reload');
+  });
+
+  shipit.blTask('server:reload', async () => {
+    await shipit.remote(`npm run server:reload`, {
+      cwd: `${process.env.DEPLOY_SERVICE_PATH}/current`,
```

Comment:
May I know what's the `current` path is used for? It seems the `current` is not existed on the existing folder structure, right?

Reply:
Does this change relate to the `cwd` settings you defined in `pm2.config.js`?

Reply:
Sorry, my bad. I just found out the `current` path is for solving [Capistrano like deployments](https://pm2.keymetrics.io/docs/tutorials/capistrano-like-deployments). So better to add this reference as a code comment.

Reply:
This whole system is a bit complicated to be explained inside code comments, I would say we need a separate document covering it, which is on my schedule.